data/net/minecraft/util/Mth.mapping

```diff
@@ -190,6 +233,16 @@ CLASS net/minecraft/util/Mth
 		ARG 0 random
 		ARG 1 minimum
 		ARG 2 maximum
+	METHOD normal (Ljava/util/Random;FF)F
+		COMMENT Generates a value from a normal distribution with the given mean and deviation.
+		ARG 0 random
+		ARG 1 mean
+		ARG 2 deviation
+	METHOD positiveCeilDiv (II)I
+		COMMENT Equivalent of {@link java.lang.Math#floorDiv(int, int)} that ceils instead of flooring.
+		COMMENT Returns the smallest (closest to negative infinity) int value that is greater than or equal to the algebraic quotient.
```

Comment:
These two sentences feel like they are switched, as the second one is more apt as the summary/first sentence of the javadoc.